backfill.py

```diff
@@ -247,23 +251,23 @@ def main():
 
     validate_args(args)
 
-    if args.type == "nightly":
+    if args.build_source == "tasklusterNightly":
```

Comment:
nitpick (blocking): fix the reference here too (maybe this should be a constant)